frontend/src/pages/answer/Answer.jsx

```diff
@@ -192,13 +216,98 @@ export default function Answer() {
     event.target.value = '';
   }
 
+  /**
+   * Save an uploaded answer to Robokache
+   */
+  async function saveAnswer() {
+    let accessToken;
+    if (isAuthenticated) {
+      try {
+        accessToken = await getAccessTokenSilently();
+      } catch (err) {
+        displayAlert('error', `Failed to validate user. Error: ${err}`);
+      }
+    } else {
+      return displayAlert('warning', 'You need to be signed in to save an answer.');
+    }
+    let response = await API.cache.createAnswer(defaultAnswer, accessToken);
+    if (response.status === 'error') {
+      return displayAlert('error', 'Failed to create answer.');
```

Comment:
Can we pass along any details to the user re: why we failed?

Reply:
Updated in 06dfb063189bcbd4f9ce23d3ace08fba1c9301f6